--- blobAndConquer-1.09/makefile~	2009-10-26 12:42:17.000000000 +0000
+++ blobAndConquer-1.09/makefile	2009-10-26 12:41:03.000000000 +0000
@@ -11,10 +11,10 @@
 DEV = 0
 
 PREFIX = $(DESTDIR)/usr
-BINDIR = $(PREFIX)/games/
-DATADIR = $(PREFIX)/share/games/blobAndConquer/
+BINDIR = $(PREFIX)/bin/
+DATADIR = $(PREFIX)/share/blobAndConquer/
 DOCDIR = $(PREFIX)/share/doc/$(PROG)/
-ICONDIR = $(PREFIX)/share/icons/
+ICONDIR = $(PREFIX)/share/icons/hicolor/
 MENUDIR = $(PREFIX)/share/applications/
 LOCALEDIR = $(PREFIX)/share/locale/
 
@@ -96,10 +96,10 @@
 
 # linking the program.
 $(PROG): $(GAMEOBJS)
-	$(CXX) $(LIBS) $(GAMEOBJS) -o $(PROG)
+	$(CXX) $(LDFLAGS) $(GAMEOBJS) -o $(PROG) $(LIBS)
 	
 pak: $(PAKOBJS)
-	$(CXX) $(LIBS) $(PAKOBJS) -o pak
+	$(CXX) $(LDFLAGS) $(PAKOBJS) -o pak $(LIBS)
 	
 %.mo: %.po
 	msgfmt -c -o $@ $<
