--- blobAndConquer-1.06/icons/blobAndConquer.desktop~	2009-07-01 14:42:34.000000000 +0200
+++ blobAndConquer-1.06/icons/blobAndConquer.desktop	2009-07-01 14:42:34.000000000 +0200
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Version=0.7
-Encoding=UTF-8
 Name=Blob Wars : Blob And Conquer
-GenericName=Game
 Comment=Mission and Objective based 3D Action Game
+Comment[pl]=Trójwymiarowa gra akcji oparta na wykonywaniu misji i zadań
 Type=Application
 Exec=blobAndConquer
 Icon=blobAndConquer
 Categories=Game;ArcadeGame;
+# vi: encoding=utf-8
